src/server/issue_request.c

```diff
@@ -257,10 +257,9 @@ issue_to_svr(char *servern, struct batch_request *preq, void (*replyfunc)(struct
 				svrname = server_host;
 		}
 	}
-	if (comp_svraddr(pbs_server_addr, svrname) == 0)
+	if (comp_svraddr(pbs_server_addr, svrname, &svraddr) == 0)
```

Comment:
Could you turn the (pbs_net_t) 0 below to NULL?

Reply:
I don't think we can, svraddr is not an address it is of type pbs_net_t which is a long. comp_svraddr sets it to 0 when it's not able to find the IP addresses associated to svrname.

Reply:
Ahh, OK.  Before we used to do things like (char *) 0.  Vincent replaced most (if not all) of these with NULL at one point.  I just thought this was one that he missed.  Since it is a long, you can't use NULL.